files/nginx.conf

```diff
@@ -88,7 +96,7 @@ http {
       set                    $saved_redirect_location '$upstream_http_location';
       proxy_pass             $saved_redirect_location;
       proxy_cache            cache;
-      proxy_cache_key        $scheme$proxy_host$uri$request_method;
+      proxy_cache_key        CACHE_KEY;
```

Comment:
As we are only caching content addressible URIs on GET, it should be okay to only include the `$uri` variable here, as the checksum of the cached content is basically included in the `$uri`. But we should then check, that we are really only cache such responses.

Reply:
:+1: